Dockerfile.dev

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.6.5-alpine
+FROM ruby:2.7-alpine
```

Comment:
This should be 2.7.1 to match the tool version.

Reply:
Do we plan to update to 2.7.* or we'll jump straight to 3? I assumed the first. Leaving the version in docker at `2.7` could help identify problems with future ruby patch releases early.

Reply:
We probably will be following patch releases, but shouldn't we be matching those to the tool-version in use so we don't get any weird differences between our local machines and the container?

Reply:
yes, we should but that's a separate thing. The pin to 2.7 in the jenkinsfile means the build will use the latest available patch version and builds will inform us about any issues with it: failing build ( unlikely), failing tests ( unlikely ) or new deprecation warnings ( likely ). 
We can then decide whether it's worth updating to the new patch release which will require new pins in .tool-versions and couple of other places.